README.md

```diff
@@ -95,7 +95,7 @@ app][electron].
 
 * **Glue code**. We maintain a [Hubot adapter][hubot-adapter] and several
 integrations ([Phabricator][phab], [Jenkins][], [Puppet][], [Redmine][],
-and [Trello][]), plus [node.js API bindings][node], and a [full-text search
+and [Trello][]), plus [node.js API bindings][node], an [isomorphic javascript library][javascript], and a [full-text search
```

Comment:
Also, do you think there's a reason people would want to use the old node.js library over yours?